python-avd/pyavd/_eos_designs/schema/schema_fragments/underlay_ipv6.schema.yml

```
@@ -10,7 +10,8 @@ keys:
     documentation_options:
       table: fabric-settings
     description: |
-      This feature allows IPv6 underlay routing protocol with RFC5549 addresses to be used along with IPv4 advertisements as VXLAN tunnel endpoints.
-      Requires "underlay_rfc5549: true" and "loopback_ipv6_pool" under the node type settings.
+      This feature turns on IPv6 routing in the underlay if the underlay_routing_protocol is some variant of "isis".
+      It is also compatible with "ebgp" combined with "underlay_rfc5549: true" or "underlay_ipv6_numbered: true".
+      Also requires a "loopback_ipv6_pool" to be defined under the node type settings.
```

This description is not super clear. Was it even correct before? I mean we added support for ipv6 underlay without RFC5549 a while ago I think.

Also instead of ignoring in certain cases we should raise and complain about incompatible things.
"`underlay_ipv6: true` is currently not supported with `underlay_routing_protocol: ospf`".

---

Adding support for ipv6 with isis also means you need to ensure it works with the "uplink_" style as well.

---

it was not updated in a long while i think so that's what i tried to address, i will try to write a clearer doc string. That's also a good point about making it work with uplink_ style, i will check how much work it is, and maybe we need to divide it into separate PRs. And will make it raise an error if used with ospf.

---

Updated this, hopefully it's more clear now.